Makefile.am

```diff
@@ -225,3 +225,12 @@ EXTRA_DIST += \
   config/stl_hash.m4 \
   mac_build.sh \
   ola.spec
+
+# -----------------------------------------------------------------------------
+
+# Create a flake8 python linter target
+if FOUND_FLAKE8
+.PHONY : flake8
+flake8:
+	flake8
```

Comment:
Perhaps consider moving the conditional inside so `make flake8` always exists but throws an error/dies when flake8 is missing. Otherwise if we write some docs saying run `make flake8` people will get a confusing make error if it's not on their system rather than a helpful one saying to install flake8 and re-run configure.

Reply:
Done.